gecode/int/distinct.hh

```diff
@@ -331,6 +372,7 @@ namespace Gecode { namespace Int { namespace Distinct {
 
 }}}
 
+#include <gecode/int/distinct/cbs.hpp>
```

Comment:
I introduced a new file because all consistency levels use the same counting algorithm.